syntactic-paths/src/main/java/com/palantir/util/syntacticpath/Path.java

```diff
@@ -339,4 +350,23 @@ private String toStringInternal() {
     public String toString() {
         return stringRepresentation.get();
     }
+
+    public static final class PathSerializer extends JsonSerializer<Path> {
```

Comment:
can you use `valueOf(String s)` and `@JsonValue` instead? untried, but something like this:

````
@JsonValue
public String toString() {
  return stringRepresentation.get();
}

public Path valueOf(String s) {
  return new Path(s);
}
````